docs/source/modules/satellite.rst

```diff
@@ -1,44 +1,61 @@
-Satellite Module
-================
+Satellite
+=========
 
-Time Class
-----------
+.. contents:: Contents
+   :depth: 1
+   :local:
 
-The :class:`Time` class allows for flexible and convenient time specifications.
-It takes Julian data in the J2000 epoch or another epoch with an appropriate
-offset (added to the times to achieve the J200 epoch). Example usage
-of the :class:`Time` class to carry out conversions can be seen on the
-tutorials page :ref:`here <Position and Time Conversions>`.
+Satellite Overview
+------------------
 
-.. autoclass:: celest.satellite.Time
-   :members:
-   :show-inheritance:
-   :noindex:
+The satellite module provides the :class:`Satellite` class for satellite specific computations. The following sections
+will detail the functionality of the :class:`Satellite` class. Additionally, a more in depth discussion of the attitude
+and look-angle calculations will be provided.
 
-Coordinate Class
-----------------
+Satellite Class
+---------------
 
-The :class:`Coordinate` class, which inherits the :class:`Time` class, is used
-to represent a satellite's position and can be used to carry out coordinate
-transformations between different frames. The Example usage of the
-:class:`Coordinate` class to carry out conversions can be seen in the tutorials
-page :ref:`here <Position and Time Conversions>`.
+The :class:`Satellite` class represents an Earth observation satellite and allows for satellite specific computations.
 
-.. autoclass:: celest.satellite.Coordinate
-   :members:
-   :show-inheritance:
-   :noindex:
+.. note::
+   The :class:`Satellite.attitude()` method requires both satellite position and velocity to be defined.
 
-Satellite Class
----------------
+The :class:`Satellite` class can be imported via the following:
+
+.. code-block:: python
 
-The :class:`Satellite` class, which inherits the :class:`Coordinate` class, is
-used to hold all necessary information for a given satellite such that it can
-be passed into the :py:func:window.generate` function for mission planning.
-Example usage of the :class:`Satellite` in a mission planning capacity can be
-seen on the tutorials page :ref:`here <Window Generation Workflow>`.
+   from celest.satellite import Satellite
 
 .. autoclass:: celest.satellite.Satellite
    :members:
-   :show-inheritance:
    :noindex:
+
+Satellite Attitude Definition
+-----------------------------
+
+Defining the satellite's attitude requires the satellite's initial and final orientations in a common frame: the
```

Comment:
This does not really define what the attitude is. Note how it represents the orientation of the satellite using rotations from some known frame.